content/javascript/concepts/objects/terms/defineProperties/defineProperties.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: '.defineProperies()'
+Description: 'Defines new or modify existing properties directly on an object'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `Object.defineProperties()` is used to define new or modify existing properties directly on an object or to add multiple properties to an object at once. This method is often used for property descriptors and fine-grained control over object properties.
+
+## Syntax
+
+```pseudo
+Object.defineProperties(object, descriptors)
+```
+
+- `object`: The object on which to define or modify it's properties.
+- `descriptors`: An object where each key represents a property name, and the corresponding value is the property descriptor.
```

Comment:
```suggestion
- `descriptors`: An object where each key represents a property name, and the corresponding value is the property descriptor.

[optional keys of descriptors]
```
`descriptors` has optional keys `value` and `writable` that you've used in the example below. Write about these keys in points. There are also keys `get` and `set` if I'm not wrong.